app-misc/lirc/lirc-0.9.4a-r2.ebuild

```diff
@@ -22,7 +22,7 @@ fi
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
-IUSE="doc static-libs X audio irman ftdi inputlirc iguanair systemd usb"
```

Comment:
Excuse my misleading comments then. `USE="static-libs"` is perfectly fine, and given that it installs .pc files, it likely documents its private dependencies (those required when linking using the static libraries) correctly. 

`prune_libtool_files` is a function from `eutils.eclass` that in conjunction with `--modules` tries to determine whether `.la` files can be removed under certain circumstances (namely - I think - if they're plugins/convenience libraries or otherwise used in a way that such libraries are not meant to be linked to).

I just recommended you ditch `prune_libtool_files` because `eutils.eclass` has grown into a monster of complex, hairy functions (hence why I also prefer users using `default` + `PATCHES` instead of `epatch` 😉 ). It's up to you if you want to put `USE="static-libs"` back (Gentoo policy is to add it only on request). Given that it was already there, I would just retain it. You're the maintainer though 😄